2022q2/core.adoc

```diff
@@ -0,0 +1,26 @@
+=== FreeBSD Core Team
+
+Contact: FreeBSD Core Team <core@FreeBSD.org>
+
+The FreeBSD Core Team is the governing body of FreeBSD.
+
+The twelfth FreeBSD Core Team was elected by active developers.
+The core.12 members are:
+
+* Baptiste Daroussin (bapt, incumbent)
+* Benedict Reuschling (bcr)
+* Ed Maste (emaste, incumbent)
+* Greg Lehey (grog)
+* John Baldwin (jhb)
+* Li-Wen Hsu (lwhsu)
+* Emmanuel Vadot (manu)
+* Tobias C. Berner (tcberner)
+* Mateusz Piotrowski (0mp)
+
+On June 10th the outgoing core.11 and incoming core.12 teams held a hand-over meeting,
+and the new Core Team is link:https://lists.freebsd.org/archives/freebsd-announce/2022-June/000034.html[announced on Jun 18].
+
+The current Core Team secretary, Muhammad Moinur Rahman (bofh), will step down
+after the appointment of new Core Team secretary and handover tasks completes.
+
+In this quarter, commit bits of Kornel Dulęba (kd) and Dmitry Salychev (dsl) have been approved.
```

Comment:
s/commit/src commit/ maybe? Not everyone may be familiar with who approves for which repo.